src/components/sidebarContent/lists/SidebarLists.vue

```diff
@@ -56,14 +56,19 @@ export default {
       const source = this.$config.axios.CancelToken.source();
 
       this.listRequests.push(source);
-      this._fetchListById({ id, cancelToken: source.token })
-        .finally(() => {
-          const index = this.listRequests.findIndex(request => request === source);
 
-          this.listRequests.splice(index, 1);
-          this.isRequestProcessing = false;
-          this._resetCustomView();
-        });
+      if (this.$route.name !== 'list') {
+        this.$router.push({ name: 'list', params: { id } });
```

Comment:
why is it here? why this situation (when this.$route.name !== 'list' is possible at all if this component is rendered)?

Reply:
MainList is rendered when route name is 'list'
SingleItem is for route 'item'

this component is rendered for both routes as both of them can have ?sidebar=lists

Reply:
without this piece of code, it was not possible to choose another list from the sidebar if you loaded one item view

Reply:
ok, understood